airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeS3StreamCopier.java

```diff
@@ -44,7 +50,7 @@ public SnowflakeS3StreamCopier(String stagingFolder,
                                  S3Config s3Config,
                                  ExtendedNameTransformer nameTransformer,
                                  SqlOperations sqlOperations) {
-    super(stagingFolder, destSyncMode, schema, streamName, client, db, s3Config, nameTransformer, sqlOperations);
+    super(stagingFolder, destSyncMode, schema, streamName,  Strings.addRandomSuffix("", "", 3) + streamName, client, db, s3Config, nameTransformer, sqlOperations);
```

Comment:
Should we add a _ or other separator between the random suffix so it's easier to read

Reply:
Yes, sure. I have already added these changes